pycaret/datasets.py

```diff
@@ -57,31 +78,41 @@ def get_data(
     Raises
     ------
     ImportError
-        When trying to import time series datasets that require sktime,
+        (1) When trying to import time series datasets that require sktime,
         but sktime has not been installed.
+        (2) If the data does not exist
     """
 
     import pandas as pd
     import os.path
     from IPython.display import display
 
-    root = address
-    data_dir = "data/"
-    meta_dir = "meta/"
+    root = (
+        "https://raw.githubusercontent.com/pycaret/datasets/main/"
+        if address is None
+        else address
+    )
+    data_dir, meta_dir = "data/", "meta/"
 
     extension = ".csv"
+    filename = str(dataset) + extension
 
-    if index:
-        filename = "index.csv"
+    folder = "common" if folder is None else folder
+
+    if dataset == "index":
         complete_address = root + meta_dir + folder + "/" + filename
```

Comment:
use os.path.join?

Reply:
It is a URL. Sos os.path.join does not work. For example, I get this 
`https://raw.githubusercontent.com/pycaret/datasets/main/data\time_series/seasonal\1.csv` as the URL and it fails to retrieve it.